bears/vcs/git/GitCommitBear.py

```diff
@@ -197,6 +202,34 @@ def check_body(self, body,
                                'HEAD commit. Please add one.')
             return
 
+        # Git keywords for closing issues
+        # https://help.github.com/articles/closing-issues-via-commit-messages/
+        git_keys = """close closes closed
```

Comment:
Also note that these are *GitHub* key words.  git does not know about them.

1. investigate whether the gitlab has the same keywords
2. rename the variable to `github_...` if the set is unique to github